packages/pyxis-react/src/components/ButtonCard/ButtonCard.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { AnchorHTMLAttributes, ButtonHTMLAttributes, FC } from 'react';
+import { IconProps } from 'components/Icon';
+import classNames from 'classnames';
+
+const getClasses = (loading:boolean, alt:boolean, className?:string):string => classNames(
+  'button-card',
+  {
+    'button-card--loading': loading,
+    'button-card--alt': alt,
+  }, className,
+);
+
+const isAnchor = (props:ButtonCardProps) => (props as CommonAnchorProps).href !== undefined;
+
+const ButtonCardContent:FC<ContentProps> = ({
+  children,
+  icon: Icon,
+  subtitle,
+}) => (
+  <>
+    <span className="button-card__icon">
+      {Icon && <Icon size="l" />}
+    </span>
+    <span className="button-card__wrapper">
+      <span className="button-card__title">
+        {children}
+      </span>
+      {subtitle && (
+      <span className="button-card__subtitle">
+        {subtitle}
+      </span>
+      )}
+    </span>
+  </>
+);
+
+const ButtonCard:FC<ButtonCardProps> = (props: ButtonCardProps) => {
+  const {
+    alt = false,
+    children,
+    className,
+    icon,
+    id,
+    loading = false,
+    subtitle,
+    tabIndex,
+    type = 'button',
+    ...restProps
+  } = props;
+
+  const Content = () => (
+    <ButtonCardContent
+      icon={icon}
+      subtitle={subtitle}
+    >
+      {children}
+    </ButtonCardContent>
+  );
+
+  if (isAnchor(props)) {
+    return (
+      <a
+        className={getClasses(loading, alt, className)}
+        data-testid={id}
+        id={id}
+        tabIndex={loading ? -1 : tabIndex}
+        {...restProps as AnchorHTMLAttributes<HTMLAnchorElement>}
+      >
+        <Content />
```

Comment:
```suggestion
        <ButtonCardContent icon={icon} subtitle={subtitle} >{children}</ButtonCardContent>
```
Misembra più chiaro così che a richiamare la funziona sopra. così anche sotto